client/src/components/map/controls/basemap/component.tsx

```diff
@@ -0,0 +1,47 @@
+import Tooltip from 'components/tooltip';
+import Image from 'next/image';
+import { useCallback } from 'react';
+
+import SatelliteImage from './images/satellite1.png';
+import DefaultImage from './images/default1.png';
+
+export type BasemapValue = 'satellite' | 'default';
+
+interface BasemapControlProps {
+  value: BasemapValue;
+  onChange: (value: BasemapValue) => void;
+}
+
+const BasemapControl: React.FC<BasemapControlProps> = ({ value, onChange }) => {
+  const handleClick = useCallback(() => {
+    if (value === 'default') {
+      onChange('satellite');
+    } else {
+      onChange('default');
+    }
+  }, [onChange, value]);
+
+  const image = value === 'default' ? SatelliteImage : DefaultImage;
+
+  return (
+    <Tooltip
```

Comment:
Do we need a tooltip? Is it in the design??

Reply:
I just found the new design, I'll ask Andreia if we still need that tooltip

Reply:
<img width="558" alt="image" src="https://user-images.githubusercontent.com/1385934/171814477-f3b6425d-9234-4678-b5d3-219b7b40720a.png">
Can confirm. I'll change the text accordingly 😁